src/CommunityToolkit/Xamarin.CommunityToolkit/Views/SideMenuView/SideMenuView.shared.cs

```diff
@@ -318,6 +341,7 @@ void PerformUpdate(bool isAnimated = true)
 			{
 				TryUpdateShift(end, true, false);
 				SetOverlayViewInputTransparent(state);
```

Comment:
I propose to change the name of SetOverlayViewInputTransparent method and invoke **changED** event there (just to avoid code duplication).

Also, I propose to move **changING** event invocation before ```if (!isAnimated)``` check to be invoking every time.

Reply:
As suggested I have renamed `SetOverlayViewInputTransparant` to `ChangeMenuState` which I think works. If you or other reviewers have strong opinions on a different name, let's come up with the best name.

I have moved the event invocation as recommend, please take another look